mapit_mysociety_org/templates/account/signup.html

```diff
@@ -21,12 +22,17 @@
             <a href="mailto:{{ CONTACT_EMAIL }}">get in touch</a> so we
             can discuss how we can provide you with the service you require.
         </p>
+
         <form id="signup_form" method="post" action="{% url "account_signup" %}" autocapitalize="off" {% if form.is_multipart %} enctype="multipart/form-data"{% endif %}>
             {% csrf_token %}
             {% include 'account/_form_fields.html' %}
             <div class="account-form__field account-form__field--submit">
-                <button type="submit" class="btn">{% trans "Sign up" %}</button>
+                <button id="customButton" class="btn btn--primary">{% trans "Sign up" %}</button>
```

Comment:
s/username/email address/ in my comment then :) In any case, as a user I'd feel reassured if there was a message telling me that no money had yet been taken in the case that the form had errors.